### PR TITLE
[MRG+1] Mep12 showcase

### PR DESCRIPTION
**Pull request by NelleV**
refs #7206

More documentation on showcase examples.

### CODE REVIEW

File: doc/conf.py [hunk 1]
```diff
@@ -28,9 +28,8 @@
 extensions = ['matplotlib.sphinxext.mathmpl', 'sphinxext.math_symbol_table',
               'sphinx.ext.autodoc', 'matplotlib.sphinxext.only_directives',
               'sphinx.ext.doctest', 'sphinx.ext.autosummary',
-              'matplotlib.sphinxext.plot_directive',
```
Comment (tacaswell) [inline doc/conf.py]: Don't we use this in other places in the code?

Comment (NelleV) [inline doc/conf.py]: hum… This was not supposed to be commited in this branch.

Comment (NelleV) [inline doc/conf.py]: oh yeah, I screwed up :D

Comment (NelleV) [inline doc/conf.py]: It's fixed.

File: examples/showcase/firefox.py [hunk 2]
```diff
@@ -2,6 +2,8 @@
 =======
 Firefox
 =======
+
+This example shows how to create the firefox logo with Path and patches.
```
Comment (anntzer) [inline examples/showcase/firefox.py]: Firefox
"Path and Patches" or "path and patches"

File: examples/showcase/xkcd.py [hunk 3]
```diff
@@ -1,3 +1,10 @@
+"""
+====
+XKCD
+====
+
+Shows how to create xkcd-like plot.
```
Comment (anntzer) [inline examples/showcase/xkcd.py]: an xkcd-like plot

File: examples/showcase/firefox.py [hunk 4]
```diff
@@ -2,6 +2,8 @@
 =======
 Firefox
 =======
+
+This example shows how to create the firefox logo with path and patches.
```
Comment (QuLogic) [inline examples/showcase/firefox.py]: Firefox

Comment (NelleV) [inline examples/showcase/firefox.py]: Fixed.

**Comment by NelleV** [comment]
Thanks @dstansby !

**Comment by dstansby** [comment]
Backported to `2.0.0-doc` via. https://github.com/matplotlib/matplotlib/commit/f49fe39f5d870c43aa2bed45ca7c73e4da287896

**Pull request merged**